docs/modules/ROOT/pages/rpc.adoc

```diff
@@ -93,148 +93,148 @@ pub const ROTATE_ON_ERROR_CODES: [u16; 1] = [429];
 
```

Comment:
I feel that this section is now more appropriate above (under the bulletpoints of "Endpoint Management" section). The transition and the horizontal break line seems out of place. 

![Screenshot 2025-06-13 at 11 55 18](https://github.com/user-attachments/assets/15b709b7-6307-43ed-a176-4e45565be828)